modules/serverless-blue-green-deploy.adoc

```diff
@@ -0,0 +1,102 @@
+[id="serverless-blue-green-deploy_{context}"]
+= Routing and managing traffic by using a blue-green deployment strategy
+
+You can safely reroute traffic from a live version of an app to a new version, by using a link:https://en.wikipedia.org/wiki/Blue-green_deployment[blue-green deployment strategy].
+
+.Procedure
+
+. Create and deploy an app as a Knative service.
+. Find the name of the first revision that was created when you deployed the service, by viewing the output from the following command:
++
+[source,terminal]
+----
+$ oc get ksvc <service_name> -o=jsonpath='{.status.latestCreatedRevisionName}'
+----
+. Add the following YAML to the service `spec` to send inbound traffic to the revision:
++
+[source,yaml]
+----
+...
+spec:
+  traffic:
+    - revisionName: <first_revision_name>
+      percent: 100 # All traffic goes to this revision
+...
+----
+. Verify that you can view your app at the URL output you get from running the following command:
++
+[source,terminal]
+----
+$ oc get ksvc <service_name>
+----
+. Deploy a second revision of your app by modifying at least one field in the `template` spec of the service. For example, you can modify the `image` of the service, or an `env` environment variable.
```

Comment:
I'm thinking if this step and the next one are redundant. I might be wrong but, they look similar and it seems to me just one or another are enough to get revision updated.